src/Miunie.Configuration/ConigurationFileEditor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Configuration;
+
+namespace Miunie.Configuration
+{
+    class ConfigurationFileEditor
+    {
+        private readonly System.Configuration.Configuration file;
+
+        internal ConfigurationFileEditor()
+        {
+            file = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+        }
+
+        internal void Save()
+        {
+            file.Save(ConfigurationSaveMode.Modified);
+            ConfigurationManager.RefreshSection(file.AppSettings.SectionInformation.Name);
+        }
+
+        internal void WriteSetting(string key, string value)
```

Comment:
What if we make it compacter by doing:

```
internal void WriteSetting(string key, string value)
        {
            if (SettingExists(key)) { UpdateSetting(key, value); }
            else { CreateSetting(key, value); }
```

Reply:
According to the style guide we tend to follow, you shouldn't collapse if statements that have an else branch.

Reply:
Right. Sorry about that.